lib/pull_request.rb

```diff
@@ -146,4 +154,18 @@ def tell_dependency_manager_what_dependabot_is_changing
       dependency_manager.add_dependency(name:, version:) if mentioned_dependencies[name]&.fetch(:to_version) == version
     end
   end
+
+private
+
+  def ci_workflow_run_id
```

Comment:
Do we need to return twice in this method?

Reply:
It can probably be tidied up 👍 will consider a small refactor PR.

Reply:
No worries, just a nitpick! Maybe something like this would work:

```
  return @ci_workflow_run_id if @ci_workflow_run_id
  ...
  @ci_workflow_run_id = ci_workflow["id"] if ci_workflow
```